lib/improver/cli/combine.py

```diff
@@ -36,71 +36,6 @@
 from improver import cli
 
 
-def main(argv=None):
```

Comment:
(@MoseleyS You'll need to remove the `if __name__` bit at the bottom as well)

Reply:
Done